### ISSUE TITLE
Run Checker in current directory

### ISSUE THREAD

**Issue by JuliaSprenger** [INT-NIT/AnDOChecker#47]
When running the AnDOChecker in the current directory using `AnDOChecker -v .` this  leads to an Error:
```
[...]
  .../ando/engine.py", line 134, in create_nested_list_of_path
    sub = directory.split(path.name)[0]
ValueError: empty separator
```

**Comment by SylvainTakerkart** [comment]
I just checked with the new version of the checker... Running the checker with the current directory as input still gives an error!

**Comment by JuliaSprenger** [comment]
Yes, that bug was not fixed yet. I also noticed that an non existing folder is also successfully validated. This I also need to fix
- [ ] fix validation of current or higher directories
- [ ] fix validation of non-existing directory (should fail)
